src/__tests__/lmdb-real-data.test.ts

```diff
@@ -0,0 +1,10 @@
+import { LmdbCache } from '../LmdbCache';
+import { defaultCacheOptions, PruneStats } from 'warp-contracts';
+
+describe.skip('Prune real Lmdb cache. Copy the real cache to ./cache, unskip and run npm run test:real', () => {
```

Comment:
oh, this one have 'skip' - I believe the leveldb version was missing the skip :-)

Reply:
yeap, my bad